docs/modules/ROOT/pages/references/release_notes.adoc

```diff
@@ -2,6 +2,59 @@
 
 TIP: This page lists notable changes in OpenShift releases which we find important. Reading release notes for you as a service.
 
+== OpenShift 4.14
+
+OpenShift version 4.14 is available since 2023-11-06.
+This version is based on Kubernetes 1.27 and CRI-O 1.27.
+The RHCOS image now uses RHEL 9.2 packages.
+Find the release notes in the upstream documentation as https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html[OpenShift Container Platform 4.14 release notes].
+The https://www.redhat.com/en/blog/red-hat-openshift-4.14-is-now-available[Red Hat OpenShift 4.14 is now available] blog post is also a valuable resource.
+
+API deprecations::
+An API has been removed in Kubernetes 1.27.
+Before updating a cluster to OpenShift 4.14, check for usage of the following API:
+
+* `CSIStorageCapacity` Resource, `storage.k8s.io/v1beta1` migrates to `storage.k8s.io/v1`
+
+See the upstream documentation on https://docs.openshift.com/container-platform/4.14/updating/preparing_for_updates/updating-cluster-prepare.html[preparing to update to OpenShift Container Platform 4.14] for detailed instructions to check for usage of these APIs.
+If any of the APIs are used, inform the affected users and ask them to update their workloads to use the APIs indicated in the upstream documentation.
+
+
+Logging in to the CLI using a web browser::
+With OpenShift Container Platform 4.14, a new oc command-line interface (CLI) flag, `--web` is now available for the oc login command.
+
+With this enhancement, you can log in by using a web browser, so that you don't need to insert your access token into the command line.
+
+Update to HAProxy 2.6::
+With this release, OpenShift Container Platform is updated to HAProxy 2.6.
+
+New option to deploy monitoring web console plugin resources::
+With this release, the monitoring pages in the Observe section of the OpenShift Container Platform web console are deployed as a dynamic plugin. With this change, the Cluster Monitoring Operator (CMO) is now the component that deploys the OpenShift Container Platform web console monitoring plugin resources.
```

Comment:
Generally we keep documentation sources at one sentence per line. This is especially helpful when making changes to existing documentation, as the diffs are much clearer.

```suggestion
With this release, the monitoring pages in the Observe section of the OpenShift Container Platform web console are deployed as a dynamic plugin.
With this change, the Cluster Monitoring Operator (CMO) is now the component that deploys the OpenShift Container Platform web console monitoring plugin resources.
```